### ISSUE TITLE
Ireland list is empty!

### ISSUE THREAD

**Issue by nashid** [lauripiispanen/most-active-github-users-counter#180]
Ireland list is empty https://commits.top/ireland_private.html

**Comment by nashid** [comment]
Screenshot attached:

<img width="1028" alt="Screenshot 2022-11-09 at 13 41 55" src="https://user-images.githubusercontent.com/6440104/200947947-455e131c-665e-4d13-b992-6c9afba2845b.png">

**Comment by ashkulz** [comment]
Duplicate of #170, see [this comment for details](https://github.com/lauripiispanen/most-active-github-users-counter/issues/98#issuecomment-1147468878) :man_shrugging:

**Comment by nashid** [comment]
@ashkulz the list for Ireland is still empty! https://commits.top/ireland_private.html

**Comment by ashkulz** [comment]
Yeah, can't help that :man_shrugging: You can try my fork which should be up-to-date: https://committers.top/ireland_private
